src/app/interfaces/storage.interface.ts

```diff
@@ -207,3 +207,13 @@ export type DiskWipeParams = [
 export interface DiskTemperatures {
   [disk: string]: number | null;
 }
+
+export interface DiskTemperatureAgg {
+  [disk: string]: TemperatureAgg;
+}
+
+export interface TemperatureAgg {
+  min: number | 0;
+  max: number | 0;
+  avg: number | 0;
```

Comment:
I think `number | 0` is literally same as `number`. Is there a reason we can't keep it simply `number` here?

Reply:
Thanks. That should be the only number. I'll fix it.